pkg/cmd/pipelinerun/delete.go

```diff
@@ -31,7 +30,7 @@ type deleteOptions struct {
 }
```

Comment:
This struct needs to be removed

Reply:
Yes, it's unused according to CI and is why `pull-tekton-cli-build-tests` are failing:

```
I1029 11:27:21.789] pkg/cmd/pipelinerun/delete.go:28:6: `deleteOptions` is unused (deadcode)
I1029 11:27:21.789] type deleteOptions struct {
```

Reply:
Oops... I'm so sorry I didn't realize I forgot to remove this :sweat:
fixed it, thank you very much!!